release-notes/opensearch-release-notes-1.0.1.md

```diff
@@ -0,0 +1,95 @@
+# OpenSearch and Dashboards 1.0.1 Release Notes
+
+## Release Highlights
+
```

Comment:
Remove this per @elfisher thanks @abhinavGupta16

Reply:
removed

Reply:
Fixed.